### PR TITLE
Add properties for payment.processing response

### PR DESCRIPTION
**Pull request by yelena-pakumirzakova-cko**
# Description of changes in PR

[//]: # Added properties Payment response Processing

## Checklist

- [x] Added a changelog entry to the `changelog.md` file in `nas_spec`
- [ ] Contacted the Tech Docs team about any corresponding guides that need to be updated for www.checkout.com/docs

**Contributing options**: Look at [our documentation](https://checkout.atlassian.net/wiki/spaces/PD/pages/2169506663/API+ref+publication+process) for options to contribute to the API reference.

[Jira](https://checkout.atlassian.net/browse/APM-7293)

### CODE REVIEW

**Review by Enrica-Mameli-cko — COMMENTED** [review]
retrieval_reference_number description should be more generic to be applicable to Card processing and APM

**Review by Enrica-Mameli-cko — DISMISSED** [review]
retrieval_reference_number description should be more generic to be applicable to Card processing and APM

File: nas_spec/components/schemas/Payments/PaymentResponse.yaml [hunk 1]
```diff
@@ -134,6 +134,27 @@ properties:
       partner_client_token:
         type: string
         description: Token to be passed to Klarna JS client.
+      partner_payment_id:
+        type: string
+        description: "Unique identification of a payment provided by Partner"
```
Comment (gareth-forshaw-cko) [inline nas_spec/components/schemas/Payments/PaymentResponse.yaml:139]: Minor one but for the "Partner"'s I think we can use a lower-case "P", i.e., "partner". Unless there is a specific use case for the "P"?

**Review by gareth-forshaw-cko — CHANGES_REQUESTED** [review]
Minor query around the "P" and could we please can we add a changelog entry for this change?

**Review by Enrica-Mameli-cko — DISMISSED** [review]

**Review by gareth-forshaw-cko — APPROVED** [review]

**Pull request merged**